{"name": "Тест: при неправильном передаче номера заказа, запрос вернет ощибку", "status": "passed", "description": "Генерирую даные курьера, получаю его id. Принимаю заказ курьером, ввожу id курьера и несуществующий track заказа в query-параметры, проверяю код 404, и тело ответа", "attachments": [{"name": "log", "source": "b34f8682-aba3-4319-803a-b51bbe2652d5-attachment.txt", "type": "text/plain"}], "start": 1717679524700, "stop": 1717679525479, "uuid": "6d5fef4b-2dc0-4633-a4e3-232dd75acaf5", "historyId": "5e38a94ab6d36ad9837c9b631374b92a", "testCaseId": "5e38a94ab6d36ad9837c9b631374b92a", "fullName": "tests.test_assept_order.TestAsseptOrder#test_assept_order_with_incorrect_query_parameter_id_order_shows_code_400_conflict", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_assept_order"}, {"name": "subSuite", "value": "TestAsseptOrder"}, {"name": "host", "value": "DESKTOP-USF6L0H"}, {"name": "thread", "value": "1796-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_assept_order"}]}